src/index.ts

```diff
@@ -168,14 +168,17 @@ if (globalThis.PublicKeyCredential) {
       value: async () => {
         let conditionalCreate = false;
         let conditionalGet = false;
-        let hybridTransport = false;
+        let hybridTransport = undefined;
         let passkeyPlatformAuthenticator = false;
         let userVerifyingPlatformAuthenticator = false;
         let relatedOrigins = false;
         let signalAllAcceptedCredentials = false;
         let signalCurrentUserDetails = false;
         let signalUnknownCredential = false;
 
+        // @ts-ignore: We're polyfilling this, so ignore whether TS knows about this or not
+        const capabilities = PublicKeyCredential.getClientCapabilities && await PublicKeyCredential.getClientCapabilities();
+
```

Comment:
Hmm, I don't think we can do this here because part of the `if()` statement checks for `!PublicKeyCredential.getClientCapabilities`. Sure, in the case that `(isSafari && browserVer >= 17.4)` is true we could end up in here and `PublicKeyCredential.getClientCapabilities` is defined, but it feels weird to try and call the native element we're defining a polyfill for, because if the method is available then why are we trying to polyfill it?

My suggestion is to drop these three lines and simply return `undefined` for `hybridTransport` as it's a capability that the browser can't query, and so this polyfill can't say for sure (i.e. should return `undefined`)

Reply:
`passkeyPlatformAuthenticator` depends on `hybridTransport`. We can speculatively return `true` if UVPAA is `true`, but what if hybrid is available but UVPAA? This effecively means `passkeyPlatformAuthenticator` isn't available either. Do you think returning `undefined` for `passkeyPlatformAuthenticator` makes sense?

I'm interested in how Edge on Linux behaves as I suspect it at least supports hybrid, but I don't have an environment. If you do, could you check? If it's too niche, may be we can just ignore?

Reply:
So, effectively only benefit of using this polyfill would be to detect whether `conditionalCreate` and `relatedOrigins` are available. Perhaps it's still useful though.